CheatSheets/windows-cheatsheet.md

```diff
@@ -146,3 +147,21 @@
 | `Backspace` | Go back to the settings home page |
 
 **[🔼Back to Top](#table-of-contents)**
+
+## Touchpad Shortcuts
+
+| Gesture | Description |
+| ------- | ----------- |
+| `Single Tap` | Single click |
+| `Tap with two fingers` | Right click |
+| `Tap twice and drag` | Multi Select |
+| `Drag two fingers` | Scroll |
+| `Pinch` | Zoom |
+| `Drag Three fingers up` | Multitasking view |
+| `Drag Three fingers down` | Show Desktop |
+| `Drag Three fingers left/right` | Switch apps |
+| `Drag Four fingers up` | Multitasking view |
+| `Drag Four fingers down` | Show Desktop |
+| `Drag Three fingers left/right` | Switch Desktops |
```

Comment:
```suggestion
```
`You should remove these commands`. Because,
These commands are already covered by the below commands: 

| `Drag Three fingers up` | Multitasking view |
| `Drag Three fingers down` | Show Desktop |
| `Drag Three fingers left/right` | Switch apps |

Reply:
Removed!
and changed now